From 95998fa176cf3728f75ab8ff4ec6c79e47b07f1e Mon Sep 17 00:00:00 2001
From: Hui Xia <hui.xia@intel.com>
Date: Wed, 15 Aug 2018 01:03:01 -0400
Subject: [PATCH 1601/1670] media: dw9714: fix cross compile issue when PM
 disabled

The compile issue was found in m68k-allmodconfig build
which disable CONFIG_PM.

Signed-off-by: Hui Xia <hui.xia@intel.com>
---
 drivers/media/i2c/dw9714.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/dw9714.c b/drivers/media/i2c/dw9714.c
index da7b4f1..11537fe 100644
--- a/drivers/media/i2c/dw9714.c
+++ b/drivers/media/i2c/dw9714.c
@@ -321,8 +321,8 @@ static int dw9714_runtime_resume(struct device *dev)
 
 #else
 
-#define dw9714_suspend	NULL
-#define dw9714_resume	NULL
+#define dw9714_runtime_suspend	NULL
+#define dw9714_runtime_resume	NULL
 
 #endif /* CONFIG_PM */
 
-- 
2.7.4

